lms/templates/admin/base.html.jinja2

```diff
@@ -37,23 +37,6 @@
                 integrity="sha512-IOebNkvA/HZjMM7MxL0NYeLYEalloZ8ckak+NDtOViP7oiYzG5vn6WVXyrJDiJPhl4yRdmNAG49iuLmhkUdVsQ=="
                 crossorigin="anonymous"
                 referrerpolicy="no-referrer"></script>
-        <script src="https://cdn.jsdelivr.net/npm/@vizuaalog/bulmajs@0.12.2/dist/bulma.min.js"></script>
```

Comment:
Removing all the stuff added to manage them JS side. 

I reckon it was interacting wierdly with this PR. 

Of course if we went this route we'd need to adapt the AI pages as well.